src/core.ts

```diff
@@ -87,43 +105,43 @@ export default {
     Left: '-1px 2px 8px rgba(106, 115, 129, 0.22)',
     Right: '1px 0px 8px rgba(106, 115, 129, 0.22)',
     Focus: '0px 0px 0px 2px rgba(82, 138, 224, 0.8)',
-  },
+  } as BoxShadowType,
   BorderRadius: {
     Small: 2,
     Medium: 4,
     Large: 8,
     XLarge: 16,
-  },
+  } as BorderRadiusType,
   FontSize: {
-    Tiny: 10,
-    XSmall: 12,
-    Small: 14,
-    ParagraphSmall: 14,
-    Body: 16,
-    Paragraph: 16,
-    H3: 20,
-    H2: 24,
-    H1: 32,
-  },
+    Tiny: 9,
+    XSmall: 11,
+    Small: 13,
+    ParagraphSmall: 13,
+    Body: 15,
+    Paragraph: 15,
+    H3: 18,
+    H2: 23,
+    H1: 31,
+  } as FontSizeType,
   FontWeight: {
     Regular: 400,
     Semibold: 600,
     Bold: 700,
-  },
+  } as FontWeightType,
   LetterSpacing: {
     Uppercase: 0.48,
```

Comment:
Looking at the Font Issue checklist, we are going to need to add more tokens for the updated letterspacing

Reply:
Looks like you added them in the `TokenType` file, but not here.

Reply:
Merged into core and removed the separate file.